src/twisted/internet/_sslverify.py

```diff
@@ -1740,29 +1747,20 @@ def _verifyCallback(conn, cert, errno, depth, preverify_ok):
 
 
 @implementer(ICipher)
-class OpenSSLCipher(FancyEqMixin, object):
+@attr.s(frozen=True)
+class OpenSSLCipher(object):
     """
     A representation of an OpenSSL cipher.
-    """
-    compareAttributes = ('fullName',)
 
-    def __init__(self, fullName):
-        """
-        @param fullName: The full name of the cipher. For example
-            C{u"ECDHE-RSA-AES256-GCM-SHA384"}.
-        @type fullName: L{unicode}
-        """
-        self.fullName = fullName
-
-
-    def __repr__(self):
-        """
-        A runnable representation of the cipher.
-        """
-        return 'OpenSSLCipher({0!r})'.format(self.fullName)
+    @param fullName: The full name of the cipher. For example
```

Comment:
In this context it needs to be an `@ivar` not a `@param`.

Reply:
Ah, yes. Fixed.